packages/flutter/test/material/text_field_focus_test.dart

```diff
@@ -548,6 +563,63 @@ void main() {
     expect(focusNodeA.hasFocus, false);
     expect(focusNodeB.hasFocus, true);
   }, variant: TargetPlatformVariant.desktop());
+
+  testWidgets('TextFormField gets blurred when tapping outside of the widget on mobile browsers', (WidgetTester tester) async {
+    final FocusNode focusNodeA = FocusNode();
+    final FocusNode focusNodeB = FocusNode();
+    final Key key = UniqueKey();
+
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Material(
+          child: ListView(
+            children: <Widget>[
+              TextField(
+                focusNode: focusNodeA,
+              ),
+              Container(
+                key: key,
+                height: 200,
+              ),
+              TextField(
+                focusNode: focusNodeB,
+              ),
+            ],
+          ),
+        ),
+      ),
+    );
+
+    final TestGesture down1 = await tester.startGesture(tester.getCenter(find.byType(TextField).first));
```

Comment:
Could you use [`tapAt](https://master-api.flutter.dev/flutter/flutter_test/WidgetController/tapAt.html) here instead of startGesutre?

Reply:
Same below.

Reply:
I've done it.